music_assistant/controllers/streams.py

```diff
@@ -395,12 +411,21 @@ async def serve_queue_item_stream(self, request: web.Request) -> web.Response:
             # no crossfade on non-tracks
             smart_fades_mode = SmartFadesMode.DISABLED
         else:
-            smart_fades_mode = await self.mass.config.get_player_config_value(
-                queue.queue_id, CONF_SMART_FADES_MODE
+            smart_fades_mode = SmartFadesMode(
+                str(
+                    await self.mass.config.get_player_config_value(
+                        queue.queue_id, CONF_SMART_FADES_MODE
+                    )
+                    or SmartFadesMode.DISABLED
+                )
```

Comment:
```suggestion
                str(
                    await self.mass.config.get_player_config_value(
                        queue.queue_id, CONF_SMART_FADES_MODE
                    )
                )
```

The configEntry has a default value it returns, it can not be None at all.
If the entry does not exist, it will raise a Keyerror

Reply:
Fixed